src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -278,9 +278,9 @@ PARAM_DEFINE_FLOAT(FW_P_LIM_MIN, -45.0f);
  * @max 60.0
  * @decimal 1
  * @increment 0.5
- * @group FW L1 Control
+ * @group FW TECS
  */
-PARAM_DEFINE_FLOAT(FW_P_LIM_MAX, 45.0f);
+PARAM_DEFINE_FLOAT(FW_P_LIM_MAX, 30.0f);
```

Comment:
Is this only used for TECS output limiting? and not also for the attitude controller limits?

Would it make sense (if there are two separate ones) to rename the param(s) to something like:
`FW_AUTO_P_MAX` and `FW_ATT_P_MAX` ?

Reply:
Yes that would make sense. Could be a follow up if you agree, as not directly related to this PR.